testing/environments/cloud/docker_image.auto.tfvars

```diff
@@ -0,0 +1,2 @@
+docker_image_override={"elasticsearch":"docker.elastic.co/cloud-release/elasticsearch-cloud-ess","kibana":"docker.elastic.co/cloud-release/kibana-cloud","apm":"docker.elastic.co/observability-ci/elastic-agent"}
+docker_image_tag_override={"elasticsearch":"8.7.0-SNAPSHOT","kibana":"8.7.0-SNAPSHOT","apm":"8.7.0-SNAPSHOT-michal-1674052792"}
```

Comment:
Looks like this was committed by accident.

Reply:
ah